jump-leveleditor/src/main/java/com/bitdecay/jump/leveleditor/render/LevelEditor.java

```diff
@@ -47,6 +47,17 @@ public static void setAssetsFolder(String assetsPath) {
         EDITOR_ASSETS_FOLDER = ASSETS_FOLDER + "editorAssets";
     }
 
+    /**
+     * The starting path defaults to the project directory.  But if you wanted to have
+     * it start from, say, a level directory inside of your resources directory, then you
+     * would set it as "src/main/resources/level" and that would start it out at that
+     * location.
+     * @param startingDirectory path to start the file dialog at
+     */
+    public static void setFileDialogStartingDirectory(String startingDirectory){
+        FileUtils.setFileChooserWorkingDirectory(startingDirectory);
+    }
```

Comment:
This method just wraps the FileUtils method which is maybe a bad practice.  But I felt like it flowed better in the blacknickel code to have two calls to LevelEditor.setupSomethingInTheEditor as opposed to a call to FileUtils.something